<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>771</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string>图片显示区域</string>
   </property>
  </widget>
  <widget class="QLabel" name="serverStatusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器状态显示</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>监听端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>410</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>监听</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
